INSTALL.md

```diff
@@ -46,9 +46,10 @@ scst-dkms package instead of the scst package.
 For example, if you want to have dkms support for your SCST rpm install, then
 you would use the following command to make your SCST packages:
 
-    make scst-dkms-rpm
+    make rpm-dkms or make scst-dkms-rpm
 
-This make step will create rpm packages that will cause SCST to be
+The first one also builds scstadmin packages in addition to the SCST dkms
```

Comment:
Consider changing `The first one` into `make rpm-dkms` to improve clarity of these instructions.